tests/torch/test_compile_torch.py

```diff
@@ -675,6 +675,7 @@ def test_compile_brevitas_qat(
     )
 
 
+@pytest.mark.xfail
```

Comment:
we need an issue to fix that please

Reply:
the issue already exists: https://github.com/zama-ai/concrete-ml-internal/issues/4576

Reply:
yes but the rule in CML is to add a FIXME on the source code to not forget
it's more for @kcelia than for you @rudy-6-4 here, don't worry